src/s1reader/s1_reader.py

```diff
@@ -470,7 +471,9 @@ def get_track_burst_num(track_burst_num_file: str = esa_track_burst_id_file):
     return track_burst_num
 
 def burst_from_xml(annotation_path: str, orbit_path: str, tiff_path: str,
-                   iw2_annotation_path: str, open_method=open, flag_apply_eap: bool = True):
+                   iw2_annotation_path: str, open_method=open,
+                   flag_apply_eap: bool = True,
+                   flag_az_fmrate_mismatch_mitigation = True):
```

Comment:
```suggestion
                   mitigate_az_fmrate_mismatch = True):
```
nit - I do not like the verbosity of the `flag_` prefix. I think the variable names can be worded so the purpose of the variable is clear. `if mitigate_az_fmrate_mismatch:` reads as "if mitigate az fmrate mismatch then...".